apps/modernization-ui/src/alert/useAlert.tsx

```diff
@@ -22,7 +24,7 @@ type AlertProviderProps = {
     children: ReactNode;
 };
 
-function AlertProvider({ duration = 5000, children }: AlertProviderProps) {
+function AlertProvider({ duration = 5000 * 5000, children }: AlertProviderProps) {
```

Comment:
Questions:
1. Is the default display time for alerts documented anywhere? 
2. Having the default duration be `5000 * 5000` (25,000,000ms or nearly 7 hours) seems odd. Perhaps making it permanent unless a duration is specified makes more sense?

Reply:
Lol sorry, meant to revert that, i had to keep it long for testing